lighthouse-core/index.js

```diff
@@ -70,7 +70,7 @@ function generateConfig(configJson, flags) {
 
 lighthouse.generateConfig = generateConfig;
 lighthouse.getAuditList = Runner.getAuditList;
-lighthouse.traceCategories = require('./gather/driver').traceCategories;
+lighthouse.traceCategories = require('./gather/driver.js').traceCategories;
 lighthouse.Audit = require('./audits/audit.js');
```

Comment:
yikes 😬 those probably need to be fixed and tests added that these don't throw?

Reply:
> yikes 😬 those probably need to be fixed and tests added that these don't throw?

I think this is a valid path? I'm not getting the context of the 😬here :)

Reply:
hm, this is super weird in the email and on disk I see

```js
lighthouse.Audit = require('./audits/audit.js');
```

but in GitHub UI I see

```js
lighthouse.Audit = require('./audits/audit.js./audits/audit.js');
```

![image](https://user-images.githubusercontent.com/2301202/58198504-f9883200-7c93-11e9-87ea-ea26a2052260.png)

seems like a GitHub bug that needs fixing not our bug! 😆